index/src/main/scala/org/dbpedia/spotlight/db/CreateVectorStore.scala

```diff
@@ -0,0 +1,39 @@
+package org.dbpedia.spotlight.db
```

Comment:
I'm a bit surprised also why this wasn't in the first PR, but it doesn't seem like it's there. I think we should keep it here anyway, it's a very useful utility to convert vector models that come from wiki2vec without rebuilding everything else